tasks/managed/get-push-rpm-params/README.md

```diff
@@ -0,0 +1,12 @@
+# get-push-rpm-params
+
+Tekton task to get params for task(push-rpm-to-koji) of pushing konflux build rpms to koji instance.
+
+## Parameters
+
+| Name                 | Description                                                               | Optional | Default value |
+|----------------------|---------------------------------------------------------------------------|----------|---------------|
+| dataPath             | Path to the JSON string of the merged data to use in the data workspace   | No       | -             |
+
+## Changes in 0.1.0
```

Comment:
not needed

Reply:
Do you mean no need this readme or the change version?

Reply:
Ah, so we don't need change log on the first version